RockLib.Logging/LogEntry.cs

```diff
@@ -339,7 +339,9 @@ private static (Func<object, string>?, Func<object, object>?) CreateStringDictio
 
     private static IReadOnlyCollection<Action<LogEntry, object>> GetSetExtendedPropertyActions(Type type) =>
         _setExtendedPropertyActionsCache.GetOrAdd(type, t =>
-            t.GetProperties(PublicInstance).Select(GetSetExtendedPropertyAction).ToArray());
+            t.GetProperties(PublicInstance)
+            .Where(_ => _.GetIndexParameters().Length == 0)
```

Comment:
Just want to double check that we don't think that anything with parameters should be included here.

Reply:
Yes, indexers are the problem, so we need to filter them out.